### PR TITLE
feat: charts toggle

### PR DESCRIPTION
**Pull request by vbstreetz**
- Implements single<>double charts toggle
- Also, relocates footer button as per designs
- https://github.com/Synthetixio/issues/issues/156

### CODE REVIEW

File: sections/exchange/FooterCard/common.tsx [hunk 1]
```diff
@@ -49,13 +49,25 @@ export const SummaryItemValue = styled.div`
 `;
 
 export const MessageContainer = styled(GridDivCentered)<{ attached?: boolean }>`
+	display: grid;
+	-webkit-box-align: center;
```
Comment (clementbalestrat) [inline sections/exchange/FooterCard/common.tsx:53]: `-webkit` probably not needed. Also I'm not sure we should be using this? Looks pretty old school. Thoughts?

File: sections/exchange/FooterCard/common.tsx [hunk 2]
```diff
@@ -49,13 +49,25 @@ export const SummaryItemValue = styled.div`
...
+
+	/*
 	width: 100%;
 	border-radius: 1000px;
 	grid-template-columns: 1fr auto;
 	background-color: ${(props) => props.theme.colors.elderberry};
 	padding: 16px 32px;
 	max-width: 750px;
 	margin: 0 auto;
+	*/
```
Comment (clementbalestrat) [inline sections/exchange/FooterCard/common.tsx:70]: let's remove the commented block

**Review by clementbalestrat — CHANGES_REQUESTED** [review]
LGTM. A few minor comments.

File: sections/exchange/TradeCard/Charts/CombinedPriceChartCard.tsx [hunk 3]
```diff
@@ -0,0 +1,327 @@
...
+				</AbsoluteCenteredDiv>
+			</ChartBody>
+		</Container>
+	);
+};
+
+function getMinNoOfDecimals(value: number): number {
+	let decimals = 2;
+	if (value < 1) {
+		const [, afterDecimal] = value.toString().split('.'); // todo
```
Comment (JChiaramonte7) [inline sections/exchange/TradeCard/Charts/CombinedPriceChartCard.tsx:295]: is this still todo?

**Review by JChiaramonte7 — APPROVED** [review]

File: pages/exchange/[[...market]].tsx [hunk 4]
```diff
@@ -47,6 +55,24 @@ const ExchangePage = () => {
 		showNoSynthsCard: true,
 	});
 
+	const [isShowingSingleChart, setIsShowingSingleChart] = useState(true);
```
[the remark this answers is not in the document]
Comment (vbstreetz) [reply pages/exchange/[[...market]].tsx:58]: fixed.

File: sections/exchange/FooterCard/common.tsx (hunk 1 above)
Comment (vbstreetz) [reply sections/exchange/FooterCard/common.tsx:53]: fixed.

File: sections/exchange/FooterCard/common.tsx (hunk 2 above)
Comment (vbstreetz) [reply sections/exchange/FooterCard/common.tsx:70]: fixed.

File: sections/exchange/TradeCard/Charts/CombinedPriceChartCard.tsx (hunk 3 above)
Comment (vbstreetz) [reply sections/exchange/TradeCard/Charts/CombinedPriceChartCard.tsx:295]: fixed.

**Pull request merged**